app/shared/date-picker/date-picker.less

```diff
@@ -1,5 +1,9 @@
 .date-picker {
-
+  input {
+    // Width value won't be taken in account because date-picker display is flex
+    // and this element has a flex-grow, flex-shrink and flex-basis values.
```

Comment:
- [ ] When someone reads only this comment and not the commit message, it could be unclear why this width is used. I suggest mentioning here that it fixes the calendar icon position in IE and Firefox.